packages/frontend/src/components/buy/BuyNear.js

```diff
@@ -175,26 +177,33 @@ const StyledContainer = styled.div`
 
 export function BuyNear({ match, location, history }) {
     const accountId = useSelector(selectAccountId);
-    const [moonPayAvailable, setMoonPayAvailable] = useState(null);
+    const [moonPayAvailable, setMoonPayAvailable] = useState(true);
```

Comment:
It seems like Moonpay won't be available everywhere, so by default, we should assume that it is not supported and let `checkMoonPay()` figure out whether it is available or not.

Reply:
I made this change from UX standpoint, let me clarify:
It looks to me that it is better to assume by default that MoonPay **is available**. 
1) There is no checks for other providers - so they are active no matter supported they are or not.
2) For users with slow internet the delay for detecting MoonPay availability may be long enough for them to decide MoonPay is not available (which would be wrong and we want to avoid misleading users).

So, in terms of progressive enhancement I see it as favorable to only block MoonPay when we know **for sure** that user will not be able to use it.

Reply:
Really good point :) 

As long as the MoonPay Option is grayed out when isMoonPayAvalaible() returns False, then this should all be fine.